README.md

```diff
@@ -23,6 +23,7 @@ This is list of known compatible USB hubs:
 | AmazonBasics       | HU9003V1EBL                                          | 7     | 3.1 |`2109:2817`| 2018    |      |
 | AmazonBasics       | HU9002V1SBL, HU9002V1ESL                             | 10    | 3.1 |`2109:2817`| 2018    |      |
 | AmazonBasics       | HUC9002V1SBL, HUC9002V1EBL, HUC9002V1ESL             | 10    | 3.1 |`2109:2817`| 2018    |      |
+| AmazonBasics       | HUC9003V1EBL                                         | 7     | 3.1 |`2109:2817`| 2018    |      |
```

Comment:
Could you please simply add HUC9003V1EBL to HU9003V1EBL line above?
Afaik, the only difference between these hubs is that HUC9* has USB-C cable included and HU9* does not, but otherwise actual hub hardware is identical.